src/product.njk

```diff
@@ -56,29 +56,38 @@ heroLg: true
                 <h5>Ticket-Based Support</h5>
             </div>
         </div>
-        <div class="mt-2"><p class="opacity-70">(some features coming soon)</p></div>
     </div>
 </div>
 
 {% include "components/divider-flow--top-red--light.njk" %}
 
-<!-- Feature List -->
-{% set cloudFeaturesHtml %}
-<div class="max-w-full overflow-x-scroll">
-    {% include "feature_lists/cloud.njk" %}
-</div>
-{% endset -%}
-
-{% set selfFeaturesHtml %}
-<div class="max-w-full overflow-x-scroll">
-    Hello Self Managed
-</div>
-{% endset -%}
+<div class="w-full ff-bg-light pb-32">
+    <div class="m-auto max-w-4xl">
+    	<div class="text-left">
+    		<h1>Sub-Minute setup</h1>
+	    	<p>With FlowForge deploying a new Node-RED project takes less than a minute. No configuration of proxies and setting up DNS, get going in less time it takes to make a cup of tee.</p>
+    	</div>
+    	<div class="text-right">
+		<h1>Entreprise-Ready</h1>
+		<p>Stacks, templates, etc
```

Comment:
Manage the versions of Node-RED being deployed and their resources. Customise the palette of nodes that are preinstalled, provide a set of default flows, and change the look and feel of the editor.